### DISCUSSION TITLE
Is there anywhere where we can find modding tutorials for RetroMCP?

### DISCUSSION THREAD

**Discussion by matthew-isbell** [MCPHackers/RetroMCP-Java#85] [General]
Title. I am new to the minecraft modding community and am not sure where to start other than by just digging through code until I figure it out on my own. If anyone has any resources, I would appreciate it

**Comment by mudkipdev** [comment]
https://github.com/PhoenixVX/golden-age-wiki

This is a good wiki for starting out with RetroMCP and modding, though you should have some programming knowledge
